pyproject.toml

```diff
@@ -150,6 +150,7 @@ black = "^24.2.0"
 pre-commit = "^3.7.0"
 ipykernel = "^6.29.4"
 semver = "^3.0.2"
+pillow = "^10.1.0"
```

Comment:
add to requirements.txt? add above in pyproject.toml list of dependencies (sorry idk why we have all three, we should fix that)

Reply:
I purpsefully kept as a dev dependency

Going to add to requirements-dev but not requirements because not everyone wants to install this but the tests do need it to pass.